README.md

```diff
@@ -15,25 +15,23 @@ If you are interested in contributing to Fuel, see our [CONTRIBUTING.md](CONTRIB
 
 ##### System Requirements
 
-There are several system requirements including clang & llvm.
+There are several system requirements including clang.
 
 ###### MacOS
 ```bash
 brew update
-brew install openssl cmake llvm
+brew install cmake
 ```
 
 ###### Debian
 ```bash
 apt update
-apt install -y cmake pkg-config libssl-dev git gcc build-essential git clang libclang-dev llvm
+apt install -y cmake pkg-config git gcc build-essential git clang libclang-dev
```

Comment:
Dup `git`. Aside: do we actually need all of these if we first install `build-essential`? I feel like `gcc` should be included in `build-essential` intuitively without actually looking into it.
```suggestion
apt install -y cmake pkg-config gcc build-essential git clang libclang-dev
```

Reply:
based on some cursory searching it seems like you're correct.